u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -185,7 +185,9 @@ u2f.getChromeRuntimePort_ = function(callback) {
  * @private
  */
 u2f.getAuthenticatorPort_ = function(callback) {
-  callback(new u2f.WrappedAuthenticatorPort_());
+  setTimeout(function() {
```

Comment:
No idea - I just know that Authenticator app does not get launched without this change.